tests/test_service.py

```diff
@@ -423,18 +424,18 @@ def testGetUserAlert(self, mockGet):
         res = self.argus.alerts.get_user_alert(testId, testId)
         self.assertTrue(isinstance(res, Alert))
         self.assertEquals(res.to_dict(), alert_D)
-        self.assertIn((os.path.join(endpoint, "alerts"),), tuple(mockGet.call_args))
+        self.assertIn((os.path.join(endpoint, "alerts/meta"),), tuple(mockGet.call_args))
```

Comment:
Thanks for fixing this. This must have been me forgetting to run the unittests with my last change.